### PR TITLE
PDS4 template to 1.B.1.0,  change radius names. Code review request (DO NOT MERGE)

### PR DESCRIPTION
**Pull request by thareUSGS**
…request (DO NOT MERGE)

## Tasklist

 - [ ] ADD YOUR TASKS HERE
 - [ ] Add test case(s)
 - [ ] Review
 - [ ] Adjust for comments
 - [ ] All CI builds and checks have passed

## Environment

Provide environment details, if relevant:

* OS:
* Compiler:

### CODE REVIEW

File: gdal/frmts/pds/pds4dataset.cpp [hunk 1]
```diff
@@ -1152,24 +1152,59 @@ void PDS4Dataset::ReadGeoreferencing(CPLXMLNode* psProduct)
...
         double dfSemiMajor = GetLinearValue(psGeodeticModel,
-                                            "semi_major_radius");
-        // according to the spec, it seems the semi_minor_radius is
-        // considered in the equatorial plane, which is rather unusual
-        // for WKT, we want to use the polar_radius as the actual semi minor
-        // axis
+                                            "a_axis_radius");
+        // Try to grab original PDS4 radius name from previous CART 
+        // schema pre-1.B.0.0 (pre LDD version 1.9.3.0)
+        if( dfDemiMajor != nullptr )
```
Comment (thareUSGS) [inline gdal/frmts/pds/pds4dataset.cpp]: This is incorrect. Not sure how to test for not found.

Comment (rouault) [reply gdal/frmts/pds/pds4dataset.cpp]: Try `CPLGetXMLNode(pseodeticModel, "a_axis_radius") != nullptr`

File: gdal/frmts/pds/pds4dataset.cpp [hunk 2]
```diff
@@ -1152,24 +1152,59 @@ void PDS4Dataset::ReadGeoreferencing(CPLXMLNode* psProduct)
...
+        {
+            double dfSemiMajor = GetLinearValue(psGeodeticModel,
```
Comment (rouault) [inline gdal/frmts/pds/pds4dataset.cpp]: remove the variable declaration (double) here

Comment (thareUSGS) [reply gdal/frmts/pds/pds4dataset.cpp]: Got it - will update

**Comment by rouault** [comment]
Closing this PR, as it will be done per https://github.com/OSGeo/gdal/issues/1851

**Pull request closed without merging**